bins/revme/src/cmd/evmrunner.rs

```diff
@@ -99,6 +104,25 @@ impl Cmd {
             if self.state {
                 println!("State: {:#?}", out.state);
             }
+            if self.trace {
```

Comment:
this `if self.trace {` should be moved few lines above.

Reply:
Would put 102 line in `else` branch. If it is `--trace` we dont need to execute it twice.

And put line 103 println! after traces.

Reply:
@qiweii